external-wallets/wagmi-wallets/src/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const WALLETS = [
+  {
+    type: "metamask",
+    name: "MetaMask",
+  },
+  { type: "frontier", name: "Frontier Wallet" },
+];
+
+export const WIDGET_CONFIG = {
+  externalWallets: true,
+  multiWallets: false,
```

Comment:
Does "wagmi" support connecting multiple wallets simultaneously? I removed this flag and tested it. Connecting a wallet from the wallets page works properly, but connecting a wallet from external buttons causes the currently connected wallet to become disconnected.

Reply:
As far as I know, no. Multi-wallet support is not available.
Here is the playground:
https://wagmi.sh/react/guides/connect-wallet
Or some website that already uses Wagmi. They all support a single wallet connection.